manager-parent/manager-v2/src/main/java/com/redhat/service/smartevents/manager/v2/persistence/models/Bridge.java

```diff
@@ -64,7 +65,7 @@ public class Bridge extends ManagedResourceV2 {
     @Column(name = "subscription_id", nullable = false, updatable = false)
     private String subscriptionId;
 
-    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER)
```

Comment:
In general it would be better not to use `FetchType.EAGER`. For the bridges I've created a `findByIdWithConditions`. In addition to that, in this PR https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/1396 I've created a `ManagedResourceV2DAO` that forces the `BridgeDAO` and the `ProcessorDAO` to have that method (because in the workers we need to fetch the resource with all the conditions)

Reply:
Because I did not add the `ProcessorWorker` yet ;)